packages/server/src/service/alloy.ts

```diff
@@ -24,6 +24,12 @@ interface AlloyResponse {
 
 const lowercaseStatuses = allRegistrationStatus.map(x => x.toLowerCase())
 
+// Alloys only accepts YYYY-MM-DD as date format
```

Comment:
1. Let's call this `toAlloyDate` and just change the date portion?
2. Can we write a test around this (these are hard to read).